app/src/main/java/com/example/livedata/User.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.livedata
+
+import android.util.Patterns
+
+const val PASSWORD_LENGTH = 5
+
+data class User(val emailAddress: String, val password: String) {
+    val isEmailValid: Boolean
```

Comment:
I imagine you'd have to do email validation more than just when you get the User object. Probably be more useful in a Utils class. 

This if fine for now because it's only used for User class.